microservices/services/query-metric/api/src/main/java/datawave/microservice/querymetric/BaseQueryMetric.java

```diff
@@ -684,6 +684,9 @@ public int getFieldNumber(String name) {
     @XmlElement
     protected String plan = null;
     @XmlElement
+    @XmlJavaTypeAdapter(StringRangeCountMapAdapter.class)
+    protected Map<String,RangeCounts> subPlans = new HashMap<>();
```

Comment:
If you want a sorted order (for logging, display, etc), you may want to use a TreeMap